zproject/default_settings.py

```diff
@@ -358,8 +358,8 @@
 # Version number for ToS.  Change this if you want to force every
 # user to click through to re-accept terms of service before using
 # Zulip again on the web.
-TOS_VERSION: Optional[str] = None
-# Template to use when bumping TOS_VERSION to explain situation.
+TERMS_OF_SERVICE_VERSION: Optional[str] = None
+# Template to use when bumping TERMS_OF_SERVICE_VERSION to explain situation.
```

Comment:
This needs an upgrade note, in case anyone else is actually setting it -- the rename effectively unsets the version, which causes new users to not have to accept the TOS.